nsqadmin/nsqadmin.go

```diff
@@ -130,11 +131,24 @@ func New(opts *Options) *NSQAdmin {
 		}
 	}
 
+	opts.BasePath = normalizeBasePath(path.Clean(opts.BasePath))
```

Comment:
I didn't initially realize that `path.Clean()` removed the trailing slash. It's part of the normalization logic so it should probably go in `normalizeBasePath()`.